src/expandableCalendar/index.js

```diff
@@ -436,7 +437,7 @@ class ExpandableCalendar extends Component {
     // TODO: turn to TouchableOpacity with onPress that closes it
     return (
       <View style={this.style.knobContainer} pointerEvents={'none'}>
-        <View style={this.style.knob}/>
+        <View style={this.style.knob} testID={CALENDAR_KNOB}/>
```

Comment:
Please add this change to CHANGELOG.md file

Reply:
@Inbal-Tish - added